src/modem/OwlModemInformation.h

```diff
@@ -74,27 +67,6 @@ class OwlModemInformation {
    */
   int getIMEI(str *out_response);
 
-  /**
-   * Retrieve Battery Charge Levels
-   * @param out_response - str object that will point to the response buffer
-   * @return 1 on success, 0 on failure
-   */
-  int getBatteryChargeLevels(str *out_response);
```

Comment:
All of the removed functionality needs to also be deleted from the OwlCLI class in the Arduino Massive IoT SDK.